### ISSUE TITLE
[Feature Request] The ability to give Blocker events different colors depending on the calendar

### ISSUE THREAD

**Issue by anthonysmillie** [MShekow/outlook-calendar-sync#4]
So I think it would be really helpful to be able to define a colour for the blocker events in each calendar based on the calendar.

IE Blocker events in Cal A that come from meetings in Cal B would be, say, green, and Blocker evens in Cal B that stem from meetings in Cal A would be, say, red.

Makes being able to tell which events are real meetings and which are blockers at a much quicker glance than having to individually read each meeting to be able to tell, especially with a really full calendar.

Don't know if colours are a Teams specific thing, which is where I typically view my calendars, but I would adore this feature.

**Comment by anthonysmillie** [comment]
Just an update here, it seems the colours are done by categories.

**Comment by anthonysmillie** [comment]
Figured it out, initialised 2 new vars for category and updated the "Create event (SynBlocker)" events with their respective vars.

**Comment by MShekow** [comment]
Thanks for reporting it, I might add this to the code base too. When you say "initialised **2** new vars for category", do you mean one variable for calendar 1 and one for calendar 2?

**Comment by anthonysmillie** [comment]
That's right yes, I wanted different colours/categories for events from each calendar so I initialised 1 variable for each calendar as a string with the category name.

Kinda did the same with the blocker prefix as well, split it out into two variables so in Cal A blockers will have the prefix "Company B Meeting: ", and in Cal B blockers have the prefix "Company A Meeting: ".

I really love the automation and am eternally grateful that you put it together, so thank you. Calendar management has been a nightmare until now.

**Comment by pimcramer** [comment]
I'd really appreciate adding in the colour categories in the code base! I've been using your flow since v0.2, great to see how you keep developing to make it more user-friendly and feature-rich.

**Comment by MShekow** [comment]
Thanks for your praise :). I've reopened this ticket as a reminder for myself to include this feature, and I'll close it again once it is done.

**Comment by MShekow** [comment]
Implemented in v0.7 - enjoy
